### DISCUSSION TITLE
Install issues

### DISCUSSION THREAD

**Discussion by AlexPBrin** [userjack6880/Open-DMARC-Analyzer#75] [General]
cant install keep getting an error with the Db

connecting to database... success
opening file... success
→ CREATE VIEW IF NOT EXISTS report_stats AS (
→ SELECT
→ report.serial, domain, rcount, disposition, reason,
→ policy_p, policy_pct, dkimdomain, dkimresult, dkim_align,
→ spfdomain, spfresult, spf_align, mindate, maxdate
→ FROM report RIGHT JOIN rptrecord
→ ON report.serial=rptrecord.serial
→ );
performing query... failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'dmarc3456.rptrecord' doesn't exist

**Comment by userjack6880** [comment]
This was addressed under issues.
